core/common/src/DisplayStyleSettings.ts

```diff
@@ -1364,6 +1367,54 @@ export class DisplayStyle3dSettings extends DisplayStyleSettings {
     this._json3d.lights = lights.toJSON();
   }
 
+  /** Adjust the solar light direction based on a date and time at a geographic location.
+   * This replaces `this.lights` with a copy that records the time point and the computed direction.
+   * @param timePoint The time in UNIX milliseconds.
+   * @param location The geographic location, or an iModel from which the geolocation is to be obtained.
+   * @see [[sunTime]] to get the current sun time.
+   * @see [[clearSunTime]] to clear the time point.
+   * @note If the iModel is not geolocated, a location in Exton, Pennsylvania will be used to compute the light direction.

```

Comment:
maybe: "if `location` is an iModel, but it is not geolocated..."

Reply:
tweaked wording.